java/arcs/android/storage/database/AndroidSqliteDatabaseManager.kt

```diff
@@ -78,6 +48,15 @@ class AndroidSqliteDatabaseManager(
         registry.close()
     }
 
+    @Deprecated("Lifecycle parameter no longer does anything. Switch to using primary" +
+                "constructor, and ensure that close is called when appropriate.")
```

Comment:
```suggestion
    @Deprecated(
        "Lifecycle parameter no longer does anything. Switch to using primary" +
        "constructor, and ensure that close is called when appropriate."
    )
```

Also, is it feasible to provide the `replaceWith` hint/value?

Reply:
I considered using `replaceWith`, but it might be tempting to just let the IDE auto-fix it, and there's more work that potentially needs to be done (cleaning up elsewhere).

If it helps, I plan to replace these usages myself in the next day or two, and remove this. I just wanted to avoid the copybara dance of death.